ml/src/test/scala/ai/h2o/sparkling/ml/algos/H2OGridSearchTestSuite.scala

```diff
@@ -69,6 +69,17 @@ class H2OGridSearchTestSuite extends FunSuite with Matchers with SharedH2OTestCo
     testGridSearch(xgboost, hyperParams)
   }
 
+  test("H2O Grid Search DRF Pipeline") {
+    val drf = new H2ODRF()
+    val hyperParams: mutable.HashMap[String, Array[AnyRef]] = mutable.HashMap()
+    hyperParams += "_ntrees" -> Array(1, 10, 30).map(_.asInstanceOf[AnyRef])
```

Comment:
I think it would be nice if we referenced hyper parameters with spark names in future

Reply:
yes, absolutely agree that we should use arguments we use in sw, not the internal H2O ones. Can you please create JIRA?

Reply:
https://0xdata.atlassian.net/browse/SW-1608

Reply:
👍 thank you!